### ISSUE TITLE
Let customer choose the gift

### ISSUE THREAD

**Issue by tmp2000** [witrin/magento-gifts#6]
Hi,It is a great extension.
Have one question:
Is it possible that let customer choose one gift in n gifts?
Thanks.

**Comment by witrin** [comment]
Yes, you're right! Currently this is not possible. But this might be a nice feature for a upcoming release. You're welcome to describe it in more detail.

**Comment by tmp2000** [comment]
So quick response :+1: 
Image such scene: 
If one customer's cart is more than $50 , so we can let customer choose 2 gifts between 5 gifts which has set in the rule in shopping cart page.
But it would have some problems for promotion rule limition,for example, there are two rules(rule 1,rule2) matching the cart more than $50,I am not sure whether it is easy or possible to check which chosen gifts are rule 1,and which chosen gifts are rule 2 after customer choose products.
Or we can auto added random 2 gifts in 5 gifts into shopping cart.
